TotalCrossSDK/src/main/java/tc/tools/AnonymousUserData.java

```diff
@@ -86,72 +91,192 @@ private static JSONObject readJsonObject(Stream stream) throws IOException {
         }
     }
 
+    public static HttpURLConnection getConn(String url) throws IOException {
+        HttpURLConnection conn = (HttpURLConnection) new URL(url).openConnection();
+        conn.setRequestMethod("GET");
```

Comment:
I don't like this approach of setting the request method as GET and changing it after to POST.

Reply:
yeap, by default a I think it is already GET, just let me make sure and will change it